Makefile

```diff
@@ -1,24 +1,54 @@
+time=/usr/bin/time -p
 
-all: ocaml fsharp rust
+.PHONY: all run force targets ocaml fsharp rust g++ clang++
+
+APP = neighbors2
+TARGETS = $(APP)_ocaml $(APP)_rust $(APP).exe $(APP)_g++ $(APP)_clang++
+
+# Build any needed targets and run
+all: $(TARGETS) run
+
+# Build any needed targets
+build: $(TARGETS)
+
+# Force build of all targets
+force: ocaml fsharp rust g++ clang++
+
+# Run all the benchmarks
+run:
+	@getent passwd $$USER | cut -d ':' -f 5 | cut -d ',' -f 1
+	@date
+	@grep 'model name' /proc/cpuinfo|uniq
 	@echo "=== F#"
-	@time mono neighbors2.exe
+	@$(time) mono neighbors2.exe
 	@echo "=== OCaml"
-	@time ./neighbors2_ocaml
+	@$(time) ./neighbors2_ocaml
 	@echo "=== rust"
-	@time ./neighbors2_rust
+	@$(time) ./neighbors2_rust
 	@echo "=== clojure"
-	@time clojure neighbors2.clj
+	@$(time) clojure neighbors2.clj
 	@echo "=== clojure_bis"
-	@time clojure neighbors2_bis.clj
+	@$(time) clojure neighbors2_bis.clj
+	@echo "=== clojure_empty (empty program for startup overhead)"	
+	@$(time) clojure < /dev/null > /dev/null
+	@echo "=== C++ (G++)"
+	@$(time) ./neighbors2_g++
+	@echo "=== C++ (Clang/LLVM)"
+	@$(time) ./neighbors2_clang++
 
-ocaml:
-	ocamlfind ocamlopt -O3 neighbors2.ml -o neighbors2_ocaml
```

Comment:
why do you remove this flag?

Reply:
It didn't work and it didn't appear to be a documented flag. I guess it is from flambda.
Will try that.